linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/tests/correct_value_dot_dir.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# variables = var_sshd_set_keepalive=0
+# plaftorm = Red Hat Enterprise Linux 9
+
+SSHD_CONFIG="/etc/ssh/sshd_config"
+
+. "$SHARED/utilities.sh"
+
+assert_directive_in_file "$SSHD_CONFIG" ClientAliveCountMax "ClientAliveCountMax 0"
```

Comment:
To prevent issues when `ClientAliveCountMax` is already configured on tested machine, we need to perform cleanup of all files, see https://github.com/ComplianceAsCode/content/blob/master/shared/templates/sshd_lineinfile/tests/wrong_value_directory.fail.sh#L11 (same for fail scenario)

Reply:
Done, thanks.